scout/server/blueprints/variants/controllers.py

```diff
@@ -994,15 +994,15 @@ def check_form_gene_symbols(
 
         if hgnc_gene is None:
             not_found_symbols.append(hgnc_symbol)
-        elif is_clinical is False:  # research
+        elif is_clinical is False:  # research variants
             updated_hgnc_symbols.append(hgnc_symbol)
-        elif is_clinical and hgnc_gene["hgnc_id"] in clinical_hgnc_ids:
+        elif hgnc_gene["hgnc_id"] in clinical_hgnc_ids:  # clinical variants
```

Comment:
What if it is None then, eh, what if?
Sorry, looks good

Reply:
I'm going to merge before I feel tempted to modify it again! 😆